applications/GeoMechanicsApplication/custom_geometries/line_interface_geometry.h

```diff
@@ -240,6 +242,13 @@ class LineInterfaceGeometry : public Geometry<Node>
         const auto points                  = this->Points();
         const auto number_of_midline_nodes = std::size_t{points.size() / 2};
 
+        auto is_null = [](const auto& rNodePtr) { return rNodePtr.get() == nullptr; };
+        if (std::any_of(points.ptr_begin(), points.ptr_end(), is_null)) {
+            // At least one point is not defined, so the points of the mid-line can't be computed.
+            // As a result, all the mid-line points will be undefined.
+            return PointerVector<Node>{number_of_midline_nodes};
+        }
```

Comment:
Ah, we missed the .get(), nice way of checking!

Reply:
In fact, we don't need the call to `get()` to check whether or not it is a null pointer, since we can use `operator==` directly. I've removed the redundant member function call.

FYI: What we had done wrong previously, was to use `begin()` and `end()`, rather than `ptr_begin()` and `ptr_end()`.